scripts/sanitycheck

```diff
@@ -569,8 +569,10 @@ class BinaryHandler(Handler):
             else:
                 generator_cmd = "make"
             command = [generator_cmd, "-C", self.outdir, "run"]
+            workdir = None
```

Comment:
I don't think this workdir / `cwd` inconsistency between `call_make_run` versus `not` is needed.

The documentation of `make` and `ninja` both state: "-C DIR: change to DIR *before doing anything else*". So I'm pretty sure you can get rid of the -C argument and replace it with an unconditional `cwd=self.outdir`. This will make the code slightly shorter instead of slightly longer.

The only drawback is dropping `self.outdir` from `verbose("Spawning process: " + ...)` but that's easy to re-add manually (on a \n new line)

Reply:
good point, fixed